.github/workflows/build_release.yml

```
@@ -0,0 +1,77 @@
+        uses: ncipollo/release-action@v1
+        with:
+          allowUpdates: true
+          artifacts: ${{ env.ARTIFACTS }}
+          commit: ${{ inputs.sha }}
+          name: ${{ inputs.name }}
+          tag: ${{ inputs.name }}
+          body: ${{ inputs.sha_short }}
+          prerelease: true
+      - name: Download latest changelog
+        if: ${{ inputs.new_tag != '' }}
+        uses: dawidd6/action-download-artifact@v2
```

actionlint (1.7.4)

the runner of "dawidd6/action-download-artifact@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)